src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -4,21 +4,22 @@ import {
   PropertyDeclarations,
   PropertyValues,
   TemplateResult,
+  customElement,
```

Comment:
I rebased on dev, not sure why it is still showing this as if it is a new change...?

Reply:
When you rebase, make sure you rebase off the latest version of upstream

```
git fetch upstream
git rebase upstream/dev
```

Reply:
That is what I always do, perhaps I botched it